lib/kazan/models.ex

```diff
@@ -52,17 +52,14 @@ defmodule Kazan.Models do
     cond do
       kind -> {:ok, kind}
       Map.has_key?(data, "kind") and Map.has_key?(data, "apiVersion") ->
-        version = case data["apiVersion"] do
-          "v" <> v ->
```

Comment:
I changed this because it would fail if any api name started with "v"

Reply:
Ah, good spot.